tests/test_bin.py

```diff
@@ -21,6 +21,9 @@ def test_pacman(self):
             # Expected exception.
             pass
 
+        # Run game of pacman with seed value entry
+        pacman.main(['-p', 'GreedyAgent', '--null-graphics', '--seed', '1234'])
```

Comment:
Make a new method and put both the capture and pacman seeded runs in there.
These test methods are not for testing a single _component_, they are for testing a single **concept**.
In this case the concept is "seeded runs".